src/validation/userValidation.js

```diff
@@ -39,7 +39,7 @@ export default class userValidation {
         .label('last name is too short'),
       email: joi.string().email().required()
         .label('invalid email format'),
-      password: new passwordComplexity(complexityOPtions).required()
+      password: new passwordComplexity(complexityOptions).required()
         .label('invalid password format'),
       gender: joi.string().valid('male', 'female').required()
```

Comment:
The error messages are not descriptive.
For instance, the same error is being sent to the user if the password was not provided or if the format is invalid
The error should be specific. `password is required` when the password is missing `password should include...` when the format is wrong

Reply:
I'll do just that. However, this doesn't need much recoding, just a change on how descriptive the password error is. If a password isn't inputted, a label like "Password is required. It should be more than 8 characters, and should include at least a capital letter, and a number" should cover both cases in a single error message.